routes/registration.js

```diff
@@ -100,7 +101,10 @@ router.post('/', function(req, res) {
                         }).catch(error => {
                             res.status(500).json({error: errorCode.PCE006.message, code: errorCode.PCE006.code});
                         });
-                    });
+                        activityProgress.create({
+                            user_id: task.dataValues.user_id
```

Comment:
What about default values for odd one out and drag and drop here?

Reply:
@Raxerz  Default values were set while declaring the model itself in ActivityProgress.js

Reply:
Okay